merkledag.go

```diff
@@ -10,16 +10,23 @@ import (
 	bserv "github.com/ipfs/go-blockservice"
 	cid "github.com/ipfs/go-cid"
 	ipldcbor "github.com/ipfs/go-ipld-cbor"
-	ipld "github.com/ipfs/go-ipld-format"
+	format "github.com/ipfs/go-ipld-format"
+	legacy "github.com/ipfs/go-ipld-legacy"
+	dagpb "github.com/ipld/go-codec-dagpb"
+	_ "github.com/ipld/go-ipld-prime/codec/raw"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
 )
 
 // TODO: We should move these registrations elsewhere. Really, most of the IPLD
 // functionality should go in a `go-ipld` repo but that will take a lot of work
 // and design.
```

Comment:
I'd just like to reflect for a moment, with gentle amusement, that the blame on this is about 4 years old.

It took awhile, but we're getting there now :)

(We'll just probably see this whole library repo disappear at the same time as this particular block of registration calls disappears and the TODO becomes finished, heh.)